source/PlanetPanel.cpp

```diff
@@ -319,6 +319,14 @@ void PlanetPanel::CheckWarningsAndTakeOff()
 	int missionCargoToSell = cargo.MissionCargoSize() - cargo.Size();
 	// Will you have to sell something other than regular cargo?
 	int cargoToSell = -(cargo.Free() + cargo.CommoditiesSize());
+	// Have you left any unique items at the outfitter?
+	bool hasUniques = false;
```

Comment:
We're listing the unique items when selling or disowning a ship. Do we want to do the same when taking off? It might feel redundant with the notice you get when selling a ship, but it may be that an outfit was individually sold off and then the player forgot to add it back onto another ship.